src/components/forms/SegmentedToggle/index.ts

```diff
@@ -0,0 +1 @@
+export { default as SegmentedToggle } from './SegmentedToggle';
```

Comment:
Missing export of enums like in https://github.com/securityscorecard/design-system/blob/alpha/src/components/Tabs/index.ts#L1

Reply:
doing it. Also, I've just fixed the imports from tests.